packer/scripts/deploy_bake.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
This script is almost a duplicate of `deploy.sh` script (except of installation of service file) Generally duplication is bad because you need to support many lines of code so to remove duplication use need to remove dependency installation from this script because your `reddit-full` image is already based on `reddit-base` image and contains all needed dependencies